Assets/AltTester/Runtime/Communication/LiveUpdateCommunicationHandler.cs

```diff
@@ -59,13 +60,16 @@ public void Init()
                 this.OnMessage(message);
             };
         }
+        
         public void Connect()
         {
+            this.isRunning = false; 
```

Comment:
Is this correct? Shouldn't it be true?

Reply:
The connection is on, but the live update should not started until the driver sends the start command.